_topic_map.yml

```diff
@@ -1287,14 +1287,14 @@ Topics:
   Topics:
   - Name: Getting started with Helm on OpenShift Container Platform
     File: getting-started-with-helm-on-openshift-container-platform
-#- Name: tkn CLI
-#  Dir: tkn_cli
-#  Distros: openshift-enterprise,openshift-webscale,openshift-origin
-#  Topics:
+- Name: tkn CLI
```

Comment:
I know this was titled previously, but I would suggest changing it from "tkn CLI" to "Tekton CLI" to align better with the other titles.

Reply:
Tekton is the name of upstream project, OpenShift Pipelines is product name.

Reply:
Discussed with Dev and QE - we can rename it to ‘OpenShift Pipelines CLI (tkn)’.
@brandi - WDTY

Reply:
That sounds good to me :)

Reply:
Hey @bmcelvee , 

I have made the change to _topics.yaml file and have also commented the entries (  had removed commented portions  just to generate preview). 

Please review and let me know if the changes are good to merge.

Thanks :)

Reply:
@bmcelvee - I have updated this PR with the pipelines attribute filename change we did day before. Please review and let me know if this PR is good to merge. Thank you :)